### PR TITLE
improve doc, add genesis flag to solo

### PR DESCRIPTION
**Pull request by libotony**
# Description

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] New and existing E2E tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have not added any vulnerable dependencies to my code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,12 +11,11 @@
 
 ## Getting Started
 
-VechainThor is the layer 1 blockchain, highly compatible† with Ethereum, which powers the vechain ecosystem.
-VechainThor is a public blockchain that is designed for mass adoption of blockchain technology by enterprise users of
+VechainThor is the layer 1 blockchain, highly compatible with EVM<sup>*</sup>, which powers the vechain ecosystem.
+VechainThor is a public blockchain that is designed for the mass adoption of blockchain technology by enterprise users of
 all sizes and is intended to serve as a foundation for a sustainable and scalable enterprise blockchain ecosystem.
 
-- † - VechainThor is currently up-to-date with the Ethereum's `Paris` hard fork, supporting Solidity version `0.8.18`.
```
Comment (libotony) [inline README.md:18]: This one also has another option, using the github's footnotes
Screenshot 2024-04-23 at 16 28 53

File: docs/usage.md [hunk 2]
```diff
@@ -176,7 +175,6 @@ bin/thor -h
 | `--target-gas-limit`        | Target block gas limit (adaptive if set to 0) (default: 0)                                  |
 | `--pprof`                   | Turn on go-pprof                                                                            |
 | `--skip-logs`               | Skip writing event\|transfer logs (/logs API will be disabled)                              |
-| `--verify-logs`             | Verify the logs DB at startup                                                               |
```
Comment (libotony) [inline docs/usage.md:179]: Removed hidden flags because they were not intended for public advertising.

File: docs/hosting-a-node.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ ___
 | CPU       | 2 Core                | 4 Core                    |
 | RAM       | 8 GB                  | 16 GB                     |
 | Bandwidth | 10 Mbit               | 20 Mbit                   |
-| Disk      | 500 GB SSD            | 1 TB fast NVMe SSD        |
+| Disk      | 500 GB NVMe SSD            | 1 TB NVMe SSD        |
```
Comment (libotony) [inline docs/hosting-a-node.md:34]: Set masternode's disk type to all NVMe SSDs but kept public node untouched.

Comment (libotony) [reply docs/hosting-a-node.md:34]: Actually, with current recommended settings, a full node only needs ~100GB, given extra space for growing up, 300GB is enough for the minimum and we can adjust the recommend to 500GB, wdyt? @darrenvechain

**Review by darrenvechain — DISMISSED** [review]

File: docs/hosting-a-node.md (hunk 3 above)
Comment (darrenvechain) [reply docs/hosting-a-node.md:32]: We have this one from vechain stats, it says a full archive node uses ~400

https://vechainstats.com/charts/#thor-size

Comment (libotony) [reply docs/hosting-a-node.md:32]: Masternode is recommended to run in full node, not full archive node. Should we mention that as well?

Comment (leszek-vechain) [reply docs/hosting-a-node.md:32]: I guess anything which help users we should mention

Comment (darrenvechain) [reply docs/hosting-a-node.md:32]: Ya I think these recommendations came from @kgapos based on running full archive nodes

**Pull request merged**